src/index.ts

```diff
@@ -1,70 +1,10 @@
-import chalk from "chalk";
-import { cosmiconfigSync } from "cosmiconfig";
-
-import getFilePaths from "./index/getFilePaths";
-import logger from "./shared/logger";
+import { getFilePaths, parseArgs, printHelp, printVersion } from "./cli";
```

Comment:
I, personally, do not see a reason to break those functions into a separate module. The `index` file is not that many LOC and it now has less of a clear ownership.